cmd/eksctl/main.go

```diff
@@ -24,11 +24,12 @@ var rootCmd = &cobra.Command{
 }
 
 func init() {
-
 	addCommands()
 
 	rootCmd.PersistentFlags().IntVarP(&logger.Level, "verbose", "v", 3, "set log level, use 0 to silence, 4 for debugging and 5 for debugging with AWS debug logging")
 	rootCmd.PersistentFlags().BoolVarP(&logger.Color, "color", "C", true, "toggle colorized logs")
+	rootCmd.PersistentFlags().BoolVar(&logger.Timestamps, "timestamps", false, "toggle timestamps in logs")
```

Comment:
I would avoid adding more flags. We can make this part `--verbose`, disable timestamps at the default level, and enable them in higher levels.

Reply:
What level would you expect this to be apart of?

Reply:
Let me know if you want me to do this differently, I've implemented it like this:

```
logger.Timestamps = false
if logger.Level >= 4 {
  logger.Timestamps = true
}
```